blocks/bricks/attention.py

```diff
@@ -290,8 +290,8 @@ class SequenceContentAttention(GenericSequenceAttention, Initializable):
     match_dim : int
         The dimension of the match vector.
     state_transformer : :class:`.Brick`
-        A prototype for state transformations. If ``None``, the default
-        transformation from :class:`.Parallel` is used.
+        A prototype for state transformations. If ``None``,
+        a linear transformation is used.
```

Comment:
`blocks-extras` should not depend on this, for examples we only test that they are runnable so far, that is the test will pass.

But the main point is that backward compatibility was never a design goal for Blocks. For such an ever-changing field as deep learning it is not possible to maintain a backward compatible framework, I believe.